EUROPACE API Calls.postman_collection.json

```
@@ -1945,7 +1945,7 @@
 						"body": {
 							"mode": "graphql",
 							"graphql": {
-								"query": "query getVorgang($vorgangsnummer: String!) {\n    vorgang(vorgangsnummer: $vorgangsnummer) {\n        vorgangsnummer\n        datenkontext\n        letztesEreignisAm\n        kundenbetreuer {\n            partnerId\n        }\n        bearbeiter {\n            partnerId\n        }\n        vorgangsstatus {\n            status\n        }\n        leadquelle\n        eigeneVorgangsnummer\n        prioritaet\n        wiedervorlage {\n            kommentar\n            wiedervorlageAm\n        }\n        finanzbedarf {\n            finanzierungszweck\n        }\n        antragsteller1 {\n            personendaten {\n                vorname\n                nachname\n                email\n            }\n        }\n        antragsteller2 {\n            personendaten {\n                vorname\n                nachname\n                email\n            }\n        }\n        aufbewahrung {\n            grund\n            endetAm\n        }\n        antraege {\n            teilantragsnummer\n            produktanbieterantragsnummer\n            angenommenAm\n            letztesEreignisAm\n            ratenkredit {\n                produktanbieterId\n            }\n            antragstellerstatus {\n                status\n            }\n            produktanbieterstatus {\n                status\n            }\n        }\n    }\n}\n",
+								"query": "query getVorgang($vorgangsnummer: String!) {\n    vorgang(vorgangsnummer: $vorgangsnummer) {\n        vorgangsnummer\n        kundenbetreuer {\n            partnerId\n        }\n        bearbeiter {\n            partnerId\n        }\n        vorgangsstatus {\n            status\n        }\n        finanzbedarf {\n            finanzierungszweck\n        }\n        antragsteller1 {\n            personendaten {\n                vorname\n                nachname\n                email\n            }\n        }\n        antragsteller2 {\n            personendaten {\n                vorname\n                nachname\n                email\n            }\n        }\n        antraege {\n            teilantragsnummer\n            angenommenAm\n            ratenkredit {\n                produktanbieterId\n            }\n            antragstellerstatus {\n                status\n            }\n            produktanbieterstatus {\n                status\n            }\n        }\n    }\n}\n",
```

Was ist denn daran kleiner? Ich würde hier gern nur wenige Felder exemplarisch drin haben wollen und nicht die Teilantragsnummer, z.B.:

```suggestion
								"query": "query getVorgang($vorgangsnummer: String!) {\n    vorgang(vorgangsnummer: $vorgangsnummer) {\n        vorgangsnummer\n        kundenbetreuer {\n            partnerId\n        }\n        antragsteller1 {\n            personendaten {\n                vorname\n                nachname\n           }\n        }\n        antragsteller2 {\n            personendaten {\n                vorname\n                nachname\n            }\n        }\n        antraege {\n            produktanbieterstatus {\n                status\n            }\n        }\n    }\n}\n",
```

Ich habe versucht die Einrückungen beizubehalten, aber keine Garantie.